src/device-manager/python/openweave/WeaveDeviceMgr.py

```diff
@@ -1017,6 +1020,12 @@ def RegisterServicePairAccount(self, serviceId, accountId, serviceConfig, pairin
         if accountId is not None and '\x00' in accountId:
             raise ValueError("Unexpected NUL character in accountId")
 
+        if pairingToken is not None and isinstance(pairingToken, str):
```

Comment:
Reviewers: please note that this is likely an artifact of a bug in the API: `weave-device-manager` calls `RegisterServicePairAccount` with an expectation to pass in a `str` as a type to `pairingToken` and `pairingInitData`.  This is inconsistent with the convention elsewhere in the code, I'd be happy to change the API convention instead.

Reply:
As discussed off-line, I think its suitable for the pairingToken and pairingInitData arguments to accept both string and bytes, as this give the API user the most flexibility.